Liveblogging Strange Loop blog post (#228)
sourcegraph/about: blogposts/live-blogging-strange-loop-2019.md

  </a>
</p>

<p class="text-center">
  <strong>Go to the liveblog - <a href="/strange-loop-liveblog/">/strange-loop-liveblog/</a></strong>
</p>
—

-->

<div class="alert alert-info text-center">
  Want to sign up to liveblog at Strange Loop?
  <a href="https://forms.gle/egsGKSrapRJkhiWw6" class="btn btn-primary ml-3" rel="nofollow" target="_blank"><i class="fa fa-file-pdf pr-2"></i>Sign up here</a>
  </div>

<img alt="Strange Loop 2019 Banner" src="/blog/strange-loop-banner.jpg" class="h5" />

Sourcegraph is the standard developer platform for [code search](/product/code-search-navigation), [review](/product/code-review), and [automation](/product/automation) for many of the largest and most exacting technology companies. With Sourcegraph, every company has access to the same kind of tools that Google and Facebook developers use every day.

---

We are huge fans of [Strange Loop](https://thestrangeloop.com)! The theme of creativity and balance of aspirational vs. practitioner content makes it one of our favorite conferences and why we are proud to be a [bronze sponsor](https://thestrangeloop.com/sponsors.html) this year.

> We're also excited to announce that Sourcegraph and Strange Loop are partnering together to produce the first Strange Loop liveblog! We've hosted and managed liveblogs for conferences such as [GopherCon](/go) and the [GraphQL summit](/graphql), and our recipe for success involves as many attendees as possible to help and contribute. Could that be you?

Reviewer: ```suggestion
We're also excited to announce that Sourcegraph and Strange Loop are partnering together to produce the first Strange Loop liveblog! We've hosted and managed liveblogs for conferences such as [GopherCon](/go) and the [GraphQL summit](/graphql), and we love to help as many community members as possible get involved and contribute! Could that be you?
```

> Read on for what's required and why its a novel way to feel like part of the conference.

Reviewer: ```suggestion
Read on for what's involved and why it's a novel way to be part of the conference.
```

## Why liveblog?

> To liveblog a talk, is to capture and summarize the essential concepts and messages, then converting that into a written form in Markdown. The publishing of your post then makes your summary available to anyone in the world, only hours after the presentation. As a liveblogger, you get benefits of deepening your knowledge of the content presented, plus your name on a promoted artifact of Strange Loop 2019.

Reviewer: ```suggestion
To liveblog a talk is to capture and summarize the essential concepts and messages in a written form. Publishing your post then makes it available to anyone in the world, only hours after the presentation. As a liveblogger, you get to deepen your knowledge of the content presented, give back to everyone in the community who couldn't be there live, and get your name on a promoted artifact of Strange Loop 2019.
```

## How to publish a liveblog post

To create and publish a liveblog post:

1. Take notes, capturing, and summarizing the talk's key points.
1. Fork our [website repository](https://github.com/sourcegraph/about), then edit the Markdown file for your post in the `blogposts` directory.
1. Optionally, you can preview your post by following the instructions to [run the dev server locally](https://github.com/sourcegraph/about/blob/master/README.md).
1. Once ready, make a pull request.
1. A preview version of the site with your blogpost will be available in the checks section of the pull request for final confirmation.
1. A Sourcegrapher will also review your pull request, and once approved and merged, will be online in less than 5 minutes.

Once live, your post will be promoted on the [Strange Loop](https://twitter.com/strangeloop_stl) and [Sourcegraph](https://twitter.com/srcgraph) Twitter accounts, crediting you and raising your profile to our combined 20,000+ followers.

## Preparing for writing your liveblog post

> We request slides from the speakers in advance to make it easier to extract code snippets, images, etc., as well as helping you get familiar with the concepts and content.

Reviewer: This seems like a sub-bullet to, or just a second sentence on bullet #1 above, rather than an entire section.

> We request slides from the speakers in advance to make it easier to extract code snippets, images, etc., as well as helping you get familiar with the concepts and content.

Reviewer: ```suggestion
We request slides from the speakers in advance to make it easier to extract code snippets, images, etc., and help you get familiar with the concepts and content.
```
